terraform/rock.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+cat <<EOF > /etc/systemd-cloud-watch.conf
+log_group = "rock"
+EOF
+
+systemctl daemon-reload
+systemctl start systemd-cloud-watch
```

Comment:
Only logging for now. I've got to look at CodeDeploy once I've got `journald` forwarding nicely.